llvm/lib/IR/Core.cpp

```diff
@@ -3258,6 +3270,30 @@ LLVMValueRef LLVMBuildIndirectBr(LLVMBuilderRef B, LLVMValueRef Addr,
   return wrap(unwrap(B)->CreateIndirectBr(unwrap(Addr), NumDests));
 }
 
+LLVMValueRef LLVMBuildCallBr(LLVMBuilderRef B, LLVMTypeRef Ty, LLVMValueRef Fn,
+                             LLVMBasicBlockRef DefaultDest,
+                             LLVMBasicBlockRef *IndrectDests,
+                             unsigned NumIndrectDests, LLVMValueRef *Args,
+                             unsigned NumArgs, LLVMOperandBundleRef *Bundles,
+                             unsigned NumBundles, const char *Name) {
+
+  SmallVector<OperandBundleDef, 8> OBs;
+  for (auto *Bundle : ArrayRef(Bundles, NumBundles)) {
+    OperandBundleDef *OB = unwrap(Bundle);
+    OBs.push_back(*OB);
+  }
+
+  SmallVector<BasicBlock *, 8> IDs;
```

Comment:
It shouldn't be necessary to create a new vector. You should be able to do something like `ArrayRef(unwrap(IndirectDests), NumIndirectDests)`.

Reply:
I tried this, but it looks like `DEFINE_SIMPLE_CONVERSION_FUNCTIONS` only defines the single pointer conversion. Some types, e.g. `LLVMTypeRef`, also define an explicit double-pointer `wrap` and `unwrap`, but basic blocks do not afaict

Reply:
I see. In that case I think it would make sense to add the unwrap for BasicBlock**. (Maybe just as a static function inside this file?)